src/main/java/com/contentful/generator/Generator.java

```diff
@@ -144,7 +144,7 @@ JavaFile generateModel(String pkg, CMAContentType contentType, String className)
         .addAnnotation(annotateModel(contentType));
 
     for (CMAField field : contentType.getFields()) {
-      if (field.isDisabled()) {
+      if (field.isDisabled() || field.isOmitted()) {
```

Comment:
No, having a field disabled and omitted are from this exiting SDKs the same, but those are two different fields in CMA ...